### DISCUSSION TITLE
What is the string of a subway exit?

### DISCUSSION THREAD

**Discussion by FloEdelmann** [organicmaps/organicmaps#3725] [Q&A] [answered]
When setting the app language to German, subway exits are labelled with "Beenden", which translates to "exit" (verb) in English. However, the correct wording would be "Ausgang", which translates to "exit" (noun) in English.

I already had a look at https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md. I searched the code for "Beenden" and found two strings: `core_exit` and `button_exit`. Is it fine to change one of them to "Ausgang" without messing up anything else? Is one of them used exclusively for subway exits?

**Comment by biodranik** [comment, marked as answer]
Great catch, thanks! core_exit is used for subways, I will fix it in #3726
